src/components/icon/icons/calculator.jsx

```diff
@@ -0,0 +1,58 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Icon from '../icon';
+
+export default function Calculator({
+  stroke,
+  strokeWidth,
+  style: styleProp,
+  width,
+  height,
+  ...rest // eslint-disable-line comma-dangle
+}) {
+  const style = {
+    ...styleProp,
+    ...Icon.defaultProps.style,
+  };
+
+  return (
+    <svg width={width} height={width} style={style} {...rest} viewBox={`0 0 36 36`}>
+      <g stroke="none" strokeWidth={strokeWidth} fill="none" fillRule="evenodd">
```

Comment:
The `transform="translate"` statements could probably be removed from the following lines (21-23).

Reply:
They need to stay :)

Reply:
Remove the now empty <g> tags as well